docs/pipeline/triggering/PERFORMANCE_TRIGGERS.md

```diff
@@ -0,0 +1,180 @@
+# Performance Triggering
+
+## Overview
+
+The `PerformanceTrigger` is a trigger mechanism that evaluates the performance of a model over time and makes decisions based on performance-observing criteria. Unlike simple triggers, the `PerformanceTrigger` considers user selected performance metrics and can evaluate decisions based on static or dynamic performance thresholds, as well as the number of avoidable misclassifications.
+
+We define `avoidable misclassifications` as the misclassifications that could have been avoided if the model had triggered earlier. They can be see as the lost benefits of not triggering. We derive the avoidable misclassification estimates by observing two time series:
+
+1. The series of evaluation scores directly after the last executed triggers (with fresh models). We define those results as `expected performance`.
```

Comment:
What does "directly" mean exactly? One metric we get? Do you stop after e.g. 2 evaluations? How is this defined?

Reply:
`directly` in the sense of `right after`. Basically in the `inform_new_model` call of the trigger. This is done after every trigger, we DO NOT stop this after a certain number of evaluations.

Reply:
Okay, it might make sense to just say "evaluate the fresh model on the last evaluation interval", if I understand correctly that is what's happening?